src/raftstore/store/util.rs

```diff
@@ -78,6 +78,31 @@ mod tests {
 
     use super::*;
 
+    // Tests the util function `check_key_in_region`.
+    #[test]
+    fn test_check_key_in_region() {
+        let test_cases = vec![// key, region start_key, region end_key, result(Ok: true, Err: false)
```

Comment:
Comment is unnecessary if the field is declared explicitly in following for loop.